SharperUniverse.Tests/SharperUniverse.Tests.csproj

```diff
@@ -1,24 +1,18 @@
-<Project Sdk="Microsoft.NET.Sdk">
-
+﻿<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
These two look exactly identical? Any particular reason this is in here?

Reply:
Hmm, I didn't touch this. Odd. Possible that Rider removed whitespace and git diff is being strange.